docs/tutorials/using-ev3-speaker.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Using the EV3 Speaker
+group: hardware-standard
+---
+
+* Table of Contents
+{:toc}
+
+
+## Introduction
+
+The EV3 uses pulse width modulation (PWM) to drive an amplifier that produces the
+sounds that you hear through the speaker. It has two modes of operation. The first,
+we call **Tone** mode (aka **Beep** mode). In Tone mode, the PWM is set to a
+frequency in the [audible range][1]. The sound you hear is a square wave.
+The second mode is called **PCM Playback** mode. In this mode, the PWM is operated
+at a higher frequency. The high frequencies are (mostly) filtered out and we are
+able to reproduce a recorded sound.
+
+[1]: https://en.wikipedia.org/wiki/Audio_frequency
+
+
+## Tone Mode
+
+There are a couple of ways to control the speaker in Tone mode. Also worth an
+honorable mention: if you plug a USB keyboard into the EV3 and press CTRL+G,
+it will beep!
+
+
+### beep command
+
+The [beep man page][2] has all you need to know.
+
+[2]: https://manpages.debian.org/jessie/beep/beep.1.en.html
+
+Examples:
+
+```
+# beep # 440 Hz for 200 ms
+# beep -f 200 -l 500 # 220 Hz for 500 ms
+# beep -f 262 -l 180 -d 20 -r 2 \
+-n -f 392 -l 180 -d 20 -r 2 \
+-n -f 440 -l 180 -d 20 -r 2 \
+-n -f 392 -l 380 -d 20 \
+-n -f 349 -l 180 -d 20 -r 2 \
+-n -f 330 -l 180 -d 20 -r 2 \
+-n -f 294 -l 180 -d 20 -r 2 \
+-n -f 262 -l 400 # twinkle twinkle little star
+```
+
+
+### `tone` sysfs Attribute
```

Comment:
If we are going to code-format "tone" in this heading, we should do the same for `beep`